README.md

```diff
@@ -14,50 +13,20 @@ Requirements
 
 Install tenantcloud.software_common
 Install tenantcloud.software_dev
+pip3 install pexpect
```

Comment:
Where you use it?

Reply:
https://monosnap.com/file/Q1EUBqfKRRapprUx8kfOKUgmNILgFq